packages/cli/.cargo/config.toml

```diff
@@ -7,3 +7,6 @@ rustflags = ["-C", "target-feature=-crt-static"]
 
 [target.armv7-unknown-linux-gnueabihf]
 linker = "arm-linux-gnueabihf-gcc"
+
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "target-feature=+crt-static"]
```

Comment:
I would do the same for x86 and aarch64 as well, which I have been doing for create-tauri-app as well.

Tauri CLI is kinda of an entry point to tauri because of `tauri info` and `tauri init` and `tauri info` should always work so they could see what they are missing.

Reply:
ah didn't see that create-tauri-app does it as well. that's good enough for me then (in context of my initial comment)